### PR TITLE
Send a QA approved email to the engineering team

### PR DESCRIPTION
**Pull request by 50kudos**

### CODE REVIEW

File: lib/flash_flow/release/percy_client.rb [hunk 1]
```diff
@@ -32,6 +32,11 @@ def gen_pdf_diffs(output_file, build_id=nil, threshold=0.0)
         PdfDiffGenerator.new.generate(get_comparisons(build_id), output_file, threshold)
       end
 
+      def qa_approved?(sha)
```
Comment (srgoldman) [inline lib/flash_flow/release/percy_client.rb]: should sha default to nil and get set to get_latest_sha if it is nil?

Comment (50kudos) [inline lib/flash_flow/release/percy_client.rb]: I think we can go either.

File: lib/flash_flow/merge/master.rb [hunk 2]
```diff
@@ -62,14 +44,65 @@ def run
...
+          end
+
+          mark_release_success(release, @git.get_sha(@git.master_branch))
+        end
+
+        errors
+      end
+
+      def mark_release_success(release, sha)
+        release['status'] = 'Pending'
```
Comment (50kudos) [inline lib/flash_flow/merge/master.rb]: This is temporary for testing.

**Comment by bradleyjucsc** [comment]
🔢

**Pull request merged**